packages/preview-button/admin/src/components/Content/index.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Markdown } from '../Markdown';
+
+export const Content = ({ data, locale, priceData, priceWidgetLabel, title }) => {
+  const isData = Array.isArray(data);
+  return (
+    <div>
+      {title && <h1>{title}</h1>}
+      {isData &&
+        data.map(
+          (item, index) =>
+            item?.content && (
+              <Markdown key={index} priceWidgetLabel={priceWidgetLabel} locale={locale} priceData={priceData}>
+                {item.content}
+              </Markdown>
+            ),
+        )}
+    </div>
+  );
+};
+
+Content.propTypes = {
+  data: PropTypes.arrayOf(
+    PropTypes.shape({
+      content: PropTypes.string,
+    }),
+  ),
+  priceData: PropTypes.arrayOf(
+    PropTypes.shape({
+      label: PropTypes.string,
+      currency: PropTypes.oneOf(['EUR', 'USD']),
+      value: PropTypes.number,
+      uuid: PropTypes.string,
+    }),
+  ),
+  locale: PropTypes.string,
+  priceWidgetLabel: PropTypes.string,
```

Comment:
Is `priceWidgetLabel` voor de tekst "gratis"`? Dan vind ik dit een onduidelijke variabele-naam. Als dat zo is, stel ik voor: `priceZeroLabel`. Ik zou 'm ook optioneel maken, zodat in principe € 0,00 wordt getoond zonder i18n instellingen.